app/views/users/show.html.erb

```diff
@@ -1,40 +1,59 @@
-<h2>Here is a list of posts for a given user</h2>
 <div class="wrapper">
-    <div class="userNameWrapper">
-        <div><img alt="Profile P."/></div>
-        <div class="userName">
-            <h2> 
-                Name: <%= @user.name %>
-            </h2>
-            <h6>Number of posts: <%= @user.posts.count%></h6>
-        </div>
+  <div class="userNameWrapper">
+    <div><img alt="Profile P."/></div>
+    <div class="userName">
+      <h2> Name: <%= @user.name %></h2>
+      <h6>Number of posts: <%= @user.posts.count %></h6>
     </div>
-        
-    <div class="bio">
-        <h3>Bio:</h3>
-        <p><%= @user.bio %></p>
-    </div>
-    <ul>
-        <% @user.posts.first(3).each_with_index do |post, index| %>
+  </div>
+
+  <div class="bio">
+    <h3>Bio:</h3>
+    <p><%= @user.bio %></p>
+  </div>
```

Comment:
- [ ] Kindly style the user show page so that the profile picture and the user details are close to each other and centered on the screen as we have in the wireframe.

- [ ] Kindly also add the **See all posts** button that leads to the users post index page.

  ![image](https://github.com/codedit334/Blog_App/assets/36542986/f652743e-08c8-44b1-9104-f868438bcce3)

Reply:
Well resolved ✔️ 

![image](https://github.com/codedit334/Blog_App/assets/38088818/1b51e412-1d13-4a01-87ae-774cca01d8c7)